content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,569 @@
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.3: Ensure that the --DenyServiceExternalIPs is set (Manual)
+
+**Result:** 🔵 Not Applicable
+
+**Details:** When DenyServiceExternalIPs is enabled, users of the cluster may not create new Services which use externalIPs and may not add new values to externalIPs on existing Service objects. It is not enabled by default, and it is not in the enabled plugins list
+
+---
+#### 1.2.4: Ensure that the --kubelet-client-certificate and --kubelet-client-key arguments are set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.5: Ensure that the --kubelet-certificate-authority argument is set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.6: Ensure that the --authorization-mode argument is not set to AlwaysAllow (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.7: Ensure that the --authorization-mode argument includes Node (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.8: Ensure that the --authorization-mode argument includes RBAC (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.9: Ensure that the admission control plugin EventRateLimit is set (Manual)
+
+**Result:** 🔵 Not Applicable
+
+**Details:** EventRateLimit admission control plugin in in Alpha state, please see https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit
+
+---
+#### 1.2.10: Ensure that the admission control plugin AlwaysAdmit is not set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.11: Ensure that the admission control plugin AlwaysPullImages is set (Manual)
+
+**Result:** 🔵 Not Applicable
+
+**Details:** AlwaysPullImages admission control will force all images to be pulled every time, it's not efficient for all users. This can be performed by a Kyverno or OPA policy
+
+---
+#### 1.2.12: Ensure that the admission control plugin SecurityContextDeny is set if PodSecurityPolicy is not used (Manual)
+
+**Result:** 🔵 Not Applicable
+
+**Details:** SecurityContextDeny admission control plugin is deprecated as of Kubernetes 1.27: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny
+
+---
+#### 1.2.13: Ensure that the admission control plugin ServiceAccount is set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.14: Ensure that the admission control plugin NamespaceLifecycle is set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.15: Ensure that the admission control plugin NodeRestriction is set (Automated)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.16: Ensure that the --profiling argument is set to false (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.17: Ensure that the --audit-log-path argument is set (Automated)
+
+**Result:** 🟠 Configurable
+
+**Details:** Audit logging is not enabled by default, it can be configured as described here: https://docs.kubermatic.com/kubeone/v1.7/tutorials/creating-clusters-oidc/#audit-logging
```

Comment:
The URLs should be `ref` instead of absolute URL, so they link to the right version of the documentation. Here's an example: https://github.com/kubermatic/docs/blob/master/content/kubeone/main/_index.en.md?plain=1#L78